borrowing/serializers.py

```diff
@@ -0,0 +1,17 @@
+from rest_framework import serializers
+
+from borrowing.models import Borrowing
+
+
+class BorrowingSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Borrowing
+        fields = (
+            "id",
+            "book",
+            "user",
+            "borrow_date",
+            "expected_return_date",
+            "actual_return_date",
+        )
+        read_only_fields = ("actual_return_date", "borrow_date")
```

Comment:
It would be better if we betrayed just the user's e-mail and the book through the StringRelatedField:
 book = serializers.StringRelatedField()
 user_email = serializers.CharField(source="user.email", read_only=True)